python/pyproject.toml

```diff
@@ -37,6 +37,7 @@ requests-toolbelt = "^1.0.0"
 
 # Enabled via `langsmith_pyo3` extra: `pip install langsmith[langsmith_pyo3]`.
 langsmith-pyo3 = { version = "^0.1.0rc2", optional = true }
+zstandard = "^0.23.0"
```

Comment:
Looks like they have a conda project so should be good, but we may have to manually update our conda distro for the SDK
https://anaconda.org/conda-forge/langsmith
https://anaconda.org/conda-forge/zstandard

Reply:
Thanks for the heads up. Would this come up when creating a new version of the sdk?

Reply:
Ya upon release. It should be straightforward to support just noting!